src/core/Microsoft.PowerToys.Settings.UI/ViewModels/KeyboardManagerViewModel.cs

```diff
@@ -113,17 +112,17 @@ public List<KeysDataModel> RemapKeys
             }
         }
 
-        public List<KeysDataModel> RemapShortcuts
+        public List<AppSpecificKeysDataModel> RemapShortcuts
         {
             get
             {
                 if (profile != null)
                 {
-                    return profile.RemapShortcuts.GlobalRemapShortcuts;
+                    return profile.RemapShortcuts.GlobalRemapShortcuts.ConvertAll(x => new AppSpecificKeysDataModel { OriginalKeys = x.OriginalKeys, NewRemapKeys = x.NewRemapKeys, TargetApp = "All Apps" }).Concat(profile.RemapShortcuts.AppSpecificRemapShortcuts).ToList();

```

Comment:
Right now we consider it as a global shortcut only if that textbox is blank (All Apps is placeholder text so it appears only when it is blank). So typing manually even All Apps would make it an app-specific shortcut. @saahmedm do we want to filter out cases where a user types in `all apps` to be also considered as a global shortcut?

Reply:
I feel not everyone might know that it has to be left blank for all apps. Even if we choose to not add `all apps` in lower case, I think we should atleast include `All Apps` or inform the user of this decision. For me personally it seems like a placeholder text, like `start typing` in powerToys Run which needs to be replaced by me as a user.

Reply:
The reason we didn't add it as default text versus placeholder text was because if a user clears a textbox they no longer know that they would have to type "All Apps" again. Which is why leaving it blank by default was the best option.

Reply:
Have you thought about having both, ie. if all text is deleted then the placeholder text would still be there? I think if you move the target app to the first column then I would definitely think that I have to type something. I just wanted to share my thoughts on usability. I'll let you and @saahmedm take a call 😄

Reply:
so I think we can filter out so All Apps or "Allapps" is just considered global. However, I think keeping the placeholder text is fine, as it relieves a user from having to type All Apps or anything when they're not interested in app-specific

Reply:
I'll make that change

Reply:
I've changed the logic such that apart from empty textbox it also considers all case variants of "All Apps" to be for global shortcuts. @saahmedm I did not consider "Allapps" as global since it's missing a space and process names can have spaces so it wouldn't be right to do "space-insensitive" comparisons. I'll merge the PR in. If required we can make the change for moving the Target App column to the start in a separate PR.